app/models/interactive_page.rb

```diff
@@ -375,16 +375,18 @@ def self.import(page_json_object, helper=nil)
         end
       end
 
-      # First, import and cache all the embeddables.
-      page_json_object[:embeddables].each do |embed_hash|
-        embed = helper.import(embed_hash[:embeddable])
-        section = embed_hash[:section]
-        import_page.add_embeddable(embed, nil, section)
-      end
-      # Now when all the objects are created, setup references (e.g. question pointing to interactive, or
-      # one embeddable pointing to another one).
-      page_json_object[:embeddables].each do |embed_hash|
-        helper.set_references(embed_hash[:embeddable])
+      page_json_object[:sections].each do |s|
+        # First, import and cache all the embeddables.
+        s[:embeddables].each do |embed_json_obj|
+          embed = helper.import(embed_json_obj.except(:column, :position))
+          import_page.add_embeddable(embed, embed_json_obj[:position], s[:title], embed_json_obj[:column])
```

Comment:
Do all sections have unique titles in LARA2?
I'd guess that if you make a page from scratch in LARA2 the sections will not have titles. So then in this case I think all embeddables will get added to the same first section.

Can you test this making a page from scratch with 2 sections and embeddables in each section, then export and import back to LARA2.  If that works I must be missing something.